packages/react-server/src/test/utilities.ts

```diff
@@ -1,5 +1,43 @@
+import {Server} from 'http';
 import {Context} from 'koa';
+import React from 'react';
+import getPort from 'get-port';
+import request from 'supertest';
 import {KoaNextFunction} from '../types';
+import {createServer} from '../server';
+
+interface Options {
+  port?: number;
+  ip?: string;
+}
+
+const servers: Server[] = [];
+
+export async function mountAppWithServer(
+  app: React.ReactElement<any>,
+  options: Options = {},
+) {
+  const {port = await getPort(), ip = 'http://localhost'} = options;
+  const url = `${ip}:${port}`;
+
+  const server = await createServer({
+    port,
+    ip,
+    render: () => app,
+  });
+
+  servers.push(server);
+
+  return request(url)
```

Comment:
it's really weird to pull in supertest but then not expose it's API at all, we could just use `fetch` or something if we were going to work purely with a response object.

Reply:
yeah true 👍

Reply:
I don't have a reason currently to use supertest, so going to remove it. We can add it if/when it's needed down the road.